app/routes/cms/index.tsx

```diff
@@ -27,8 +27,8 @@ export default function Index(): JSX.Element {
 	const { firstname, lastname } = useLoaderData<LoaderData>();
 
 	return (
-		<Center minW="100vw" minH="100vh">
-			<Heading>
+		<Center h="100%" w="100%">
+			<Heading m={8}>
```

Comment:
Quite a lot of margin, considering this will only affect small viewports I would go for something that allows for the smallest screen size possible while still looking ok, which would be a `2` or so

Reply:
This does affect every viewport, because the parent does not have a fixed size.

Reply:
Oh, nevermind. This only affects small viewports